src/shared/install.h

```diff
@@ -35,11 +35,16 @@ enum UnitFileChangeType {
 };
 
 enum UnitFileFlags {
-        UNIT_FILE_RUNTIME = 1 << 0,
-        UNIT_FILE_FORCE   = 1 << 1,
-        UNIT_FILE_DRY_RUN = 1 << 2,
+        UNIT_FILE_RUNTIME  = 1 << 0,
+        UNIT_FILE_FORCE    = 1 << 1,
+        UNIT_FILE_DRY_RUN  = 1 << 2,
+        UNIT_FILE_PORTABLE = 1 << 3,
 };
 
+#define SD_SYSTEMD_UNIT_RUNTIME  (UINT64_C(1) << 0)
+#define SD_SYSTEMD_UNIT_FORCE    (UINT64_C(1) << 1)
+#define SD_SYSTEMD_UNIT_PORTABLE (UINT64_C(1) << 2)
```

Comment:
why these three mcros? The enum above should suffice, no?

Reply:
(i mean, reorder the enum above so that dry-run is at the end, given that we don't want to export that)

Reply:
Because of the unused one, but I guess it doesn't matter in the end - will rework

Reply:
removed